libqtile/widget/base.py

```diff
@@ -369,6 +370,35 @@ def mouse_enter(self, x, y):
     def mouse_leave(self, x, y):
         pass
 
+    def _draw_with_mirrors(self):
```

Comment:
mypy ignores methods that don't have at least one annotation so we can get it to check this one too with this:
```suggestion
    def _draw_with_mirrors(self) -> None:
```
(You know how much I love my typing! ;))